src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -490,6 +491,14 @@ export default {
 			this.handleScroll()
 			this.handleUnreadMention()
 		}, 50),
+
+		// Route to the first search result
+		handleEnter() {
+			if (this?.conversationsList[0]?.token) {
+				this.$router.push({ name: 'conversation', params: { token: this.conversationsList[0].token } })
+			}
+			this.handleClickSearchResult()
```

Comment:
This should start a 1-1 if the first shown entry is a user?
Doesn't work for me